kNN_NBC/NBC/kpnn_nbcrunner.cpp

```diff
@@ -0,0 +1,18 @@
+#include "kpnn_nbcrunner.h"
+
+
+void KpNN_NBCRunner::insertCloserPointToNeihborhood(orderedNeighborhood& nb, Point* point, double dist)
+{
+	double eps = nb.begin()->first;
+
+	int num_eq_to_first = 1;
```

Comment:
Prośba o wyjaśnienie (może być odnośnik do sekcji artykułu) co to jest za zmienna

Reply:
liczba punktów w takiej samej odległości jak najdalszy spośród obecnych kandydatów na sąsiadów

Reply:
Zmieńmy proszę na points_eq_dist_to_first (w sensie equally distant) bo tego trudno było się domyślić

Reply:
ok